ospf6d/ospf6_neighbor.c

```diff
@@ -892,6 +910,7 @@ static void ospf6_neighbor_show(struct vty *vty, struct ospf6_neighbor *on,
 		else
 			snprintf(nstate, sizeof(nstate), "DROther");
 	}
+	ospf6_neighbor_state_message(on, nstate, sizeof(nstate));
```

Comment:
I'm not sure I really understand this change at all.  This just completely blows away the nstate changes starting on line 900 and go to line 912.  what's the goal here at all?

Reply:
align the output state between ospfv2 and ospfv3.
we want to achieve it step by step.

Reply:
taken into account